<launch>
  <node pkg="leg_detector3" type="leg_detector" name="leg_detector" args="scan:=base_scan $(find leg_detector)/config/trained_leg_detector.yaml" output="screen">
  </node>
  <node pkg="dynamic_reconfigure" name="reconfigure_para" type="dynparam" args="set_from_parameters leg_detector">
	<param name="fixed_frame" type="string" value="odom"/>
  </node>
</launch>
